Makefile.include

```diff
@@ -532,7 +532,12 @@ ifeq (,$(FLASHFILE))
 endif
 
 # By default always build ELFFILE and FLASHFILE
```

Comment:
Now the default, is to build ELFFILE, BINFILE and FLASHFILE so the comment should be updated I think.

Reply:
I updated below